examples/cloud-router-2-aws-connection/variables.tf

```diff
@@ -46,7 +46,7 @@ variable "aside_fcr_uuid" {
 variable "zside_ap_authentication_key" {
   description = "Authentication key for provider based connections"
   type        = string
-  sensitive   = true
```

Comment:
This should still be `sensitive = true`. No default should be here because it's required.